src/App.tsx

```diff
@@ -1,13 +1,19 @@
-import { MantineProvider } from '@mantine/core'
+import "@mantine/core/styles.css";
+import { MantineProvider } from "@mantine/core";
+import '@mantine/dates/styles.css';
 
-function App() {
-  
+import "./index.css";
+import { TasksProvider } from "./contexts/TasksContext";
+import HomePageContainer from "./Pages/HomePage/Containers/HomePageContainer";
 
+function App() {
   return (
-    <MantineProvider>
-      App
-    </MantineProvider>
-  )
+    <TasksProvider>
+      <MantineProvider defaultColorScheme="dark">
```

Comment:
[NIT] Mantine provider does not need access to the TasksProvider context
```suggestion
    <MantineProvider defaultColorScheme="dark">
      <TasksProvider>
```